src/Components/BrowsePage/BrowsePage.js

```diff
@@ -10,7 +10,7 @@ export const BrowsePage = (props) => {
             return (
                 <li key ={i}className="hexa">
                     <div className="inner-hexagon">
-                        <img className="hexa-image" 
+                        <img className="hexa-image" data-testid='image-test' 
```

Comment:
I gotta start trying this data-testid. it looks quite handy.

Reply:
Agreed! Makes it pretty easy to target specific elements via testing.

Reply:
I agree I should implement it more from now on